doc/rados/configuration/mclock-config-ref.rst

```diff
@@ -29,16 +30,18 @@ Ceph cluster enables the throttling of the operations(IOPS) belonging to
 different client classes (background recovery, scrub, snaptrim, client op,
 osd subop)”*.
 
-The purpose of the mclock profile is to help with the following,
+The purpose of the mclock profile is to help with the following:
 
-- Based on the capacity limits provided and the mclock profile selected,
-  determine the low level mclock resource control parameters,
-- Transparently apply the lower level mclock resource control parameters and
-  some Ceph configuration parameters depending on the profile.
+- the mclock profile uses the capacity limits provided within it and the mclock
```

Comment:
- it uses the capacity limits and the mclock profile selected by the user to determine the low-level mclock resource control parameters